### PR TITLE
Post 8/3

### PR DESCRIPTION
**Pull request by csell5**
* Session Submit Form updates include, duration, submit anytime, better edit and validation
* Board sorts by day/time
* Copy updated
* help page removed until finished
* coverts off vanity dns
* my submissions returns all for user
* my favorites returns default event ( enhancements later )
* board returns today + 30 days out.
* cannot edit past sessions
* can cancel future sessions
* cancelled sessions do not appear once cancelled ( enhancements coming ) 
* perf improvements ( like only call an api once when you need too ;) )
* removed ticket purchase
* updated new signup to take you to signup rather than login

### CODE REVIEW

**Comment by brettski** [comment]
What do you mean by, " many timezones have the same offset values", or is that an old comment on this pr? Where are you seeing these offsets?

**Comment by csell5** [comment]
It could be -5 but there are loads of zones that fall into -5

**Comment by csell5** [comment]
Are we saving UTC in the dB?

**Comment by brettski** [comment]
But were are you seeing that?

**Comment by brettski** [comment]
The Database will show a Timestamp type value in your local Timezone. Only if it is a string will it show in UTC.

**Comment by csell5** [comment]
@brettski I think I have `create` and `edit` working. I haven't messed with the dashboard yet. Can you test, and dump my test sessions.

**Comment by csell5** [comment]
added issues #263 and #264 as a result of this work.

**Comment by brettski** [comment]
@csell5 Did my testing, great addition! I added a few issues based on findings.

**Comment by brettski** [comment]
@csell5 Test records removed from Datastore

**Comment by csell5** [comment]
I'm going to pull in #267 #268 and #269 to this pr too.

**Comment by csell5** [comment]
@brettski refactored the select boxes addressing a few of the issues you found. that 'should' close out #269 and #268. also added the withdraw #140 but we need to talk about the backend some and I still need to refactor the lists to better account for sessions in a non accepted status. 

can you test create and edit again?

**Comment by brettski** [comment]
Fixes #269 
Fixes #268 
140 needs some work

**Comment by brettski** [comment]
I pushed an update to the branch changing the formatting of the date in the edit form, which fixes #267 . Remove the format line if you don't like it.

**Comment by brettski** [comment]
Commit 7c923d5, changed the way dayjs parsed the date and time zone and it seems to have corrected our date issue when times of < 01:00 are used the day would subtract one when using .toDate() or toISODate().

**Comment by csell5** [comment]
@GeekOnCoffee if you want to take a gander that would be great. This should be ready to fly but I am sure I spelled something wrong ;)

File: src/dataSources/api.that.tech/sessions.js [hunk 1]
```diff
@@ -176,7 +230,15 @@ export default (client, eventId = config.eventId) => {
     return client
       .mutation(UPDATE_SESSION_BY_ID, mutationVariables)
       .toPromise()
-      .then((r) => r.data.sessions.session.update.openSpace);
+      .then((r) => {
+        if (r.error) {
+          // eslint-disable-next-line no-console
+          console.error(r.error);
```
Comment (brettski) [inline src/dataSources/api.that.tech/sessions.js:236]: Do we want this to remain in production?

File: src/elements/GroupHeader.svelte [hunk 2]
```diff
@@ -1,8 +1,17 @@
 <div>
+
+  <h2
+    class="text-3xl leading-9 font-extrabold tracking-tight text-gray-900
+    sm:text-4xl sm:leading-10"
+  >
+    <slot />
+  </h2>
+</div>
+<!-- <div>
```
Comment (brettski) [inline src/elements/GroupHeader.svelte:10]: Does this need to stay?

File: src/dataSources/api.that.tech/sessions.js (hunk 1 above)
Comment (csell5) [reply src/dataSources/api.that.tech/sessions.js:236]: yea for now. we need to get sentry in somehow but this at least will bubble up and log something such that we can debug with people.

File: src/elements/GroupHeader.svelte (hunk 2 above)
Comment (csell5) [reply src/elements/GroupHeader.svelte:10]: I was messing around with it and decided to just leave it until I come back to fixing it up. didn't want to hold this up.

File: src/config.js [hunk 3]
```diff
@@ -10,9 +9,20 @@ export const securityConfig = {
...
-  api: 'https://api.that.tech',
+  // api: 'https://api.that.tech',
+  api: 'https://gateway-bgydxslf5a-uc.a.run.app',
   profileImageApi: 'https://api.that.tech/profile',
   defaultProfileImage:
     'https://images.that.tech/members/person-placeholder.jpg?auto=format&fit=facearea&facepad=10&mask=ellipse&h=250&w=250&q=50&dpr=2',
   gtag: 'UA-21705613-11',
 };
+
+export const events = {
```
[the remark this answers is not in the document]
Comment (csell5) [reply src/config.js:20]: yea just for now. TTB was my first thought but we will just need a better way to deal with the +1 nature of events. kinda in a weird spot at the moment.

File: package.json [hunk 4]
```diff
@@ -56,28 +57,31 @@
...
+    "tailwindcss": "^1.6.2",
     "tailwindcss-font-inter": "^1.0.8",
     "yrv": "0.0.30",
-    "yup": "~0.29.1"
+    "yup": "~0.29.3"
   },
   "dependencies": {
-    "sirv-cli": "^1.0.3",
+    "sirv-cli": "^1.0.5",
     "uninstall": "0.0.0"
```
[the remark this answers is not in the document]
Comment (csell5) [reply package.json:85]: removing now.

**Pull request merged**